book/creating-a-basic-website/adding-localizations.rst

```diff
@@ -56,3 +56,39 @@ the content. In case there is a ghost page - which means that the page is not
 translated into the current localization - this tree will be used to determine
 the "closest" language available.
 
+Localizing Strings
```

Comment:
this shouldn't be in this doc-file because this file handles backend locales not website translations. Also i am not should if we should describe symfony features in our docs /cc @danrot @webmozart @chirimoya